### DISCUSSION TITLE
part-ii/2021-88/

### DISCUSSION THREAD

**Discussion by tripos-comments** [tripos-education/maths-tripos-questions#357] [Question comments]
Part II 2021 Logic and Set Theory: Paper 2, Section II, G 

https://questions.tripos.org/part-ii/2021-88/

**Comment by artie2000** [comment]
For the first parts, use ordinal induction for the proofs.

<details>
<summary>Sum of CNFs:</summary>

$$\omega^{\delta_{1}} m_{1}+\omega^{\delta_{2}} m_{2}=\begin{cases}\omega^{\delta_{1}} m_{1}+\omega^{\delta_{2}} m_{2}&\delta_1>\delta_2\\\omega^{\delta_{1}}(m_{1}+m_{2})&\delta_1=\delta_2\\\omega^{\delta_{2}} m_{2}&\delta_1<\delta_2\end{cases}$$

To show this, factor out on the left until you are left with a sum like $m_1+\omega^\gamma m_2=\omega^\gamma m_2$.

In general, the CNF of $\alpha$ is cut off in the first place where $\beta_k<\beta'_1$.

</details>
